docs/releasing.md

```diff
@@ -1,69 +1,42 @@
-3. Write the [release notes](#release-notes) and make sure the binaries uploaded return the correct version
-4. Push the docker images that were generated with this release tool
-5. Publish release
-6. [Announce][release-announcement] the release
-
-## Manual
-
-1. Tag the repository and push the tag
-   - An [OWNER](/OWNERS) runs `git tag -s <version>` and inserts the changelog and pushes the tag with `git push <version>`
-2. Create a draft release in GitHub and associate it with the tag that was just created
-3. Checkout the tag and make sure git is in a clean state
-4. Run `make release-artifacts`, with the appropriate environment variables:
-   ```
-   REGISTRY="quay.io/k8s" \
-   PULL_POLICY="IfNotPresent" \
-   MANAGER_IMAGE_TAG="<version>" \
-   make release-artifacts
-   ```
-5. Attach the tarball to the drafted release
-6. Attach `clusterctl` to the drafted release (for darwin and linux architectures)
-7.  Write the [release notes](#release-notes) and make sure the binaries uploaded return the correct version
-8.  Build and push a container image to the staging repository using the release candidate tag (`<version>`):
-    ```
-    REGISTRY="quay.io/k8s" \
-    MANAGER_IMAGE_TAG="<version>" \
-    time make docker-build
-
-    REGISTRY="quay.io/k8s" \
-    MANAGER_IMAGE_TAG="<version>" \
-    time make docker-push
-    ```
-11. Publish release
-12. [Announce][release-announcement] the release
+## Change milestone 	
+ - Change milestone applier so new changes can be applied to the appropriate lease 
+      - Open a PR in https://github.com/kubernetes/test-infra to change the following line https://github.com/kubernetes/test-infra/blob/master/config/prow/plugins.yaml#L323
+- Example PR: https://github.com/kubernetes/test-infra/pull/16827
+
+## Prepare branch, tag and release-notes	
+ - Identify a known good commit on the main branch
+ - Fast forward release branch to selected commit, always release from the is branch and not from master
+- git fetch upstream
+- git merge --ff-only upstream/master
+ - Create tag with git 
+- export RELEASE_TAG=v0.4.6 # the tag of the release to be cut
+- git tag -s ${RELEASE_TAG} -m "${RELEASE_TAG}"
+	- -s creates a signed tag, user must have GPG added to heir github count
+- git push upstream ${RELEASE_TAG}
+ `make release` from repo, this will create the release artifacts in the t/ folder 
+ Release notes https://github.com/kubernetes/release/blob/master/cmd/lease-notes/README.md
+- Export GITHUB_TOKEN
+release-notes --github-org kubernetes-sigs --github-repo uster-api-provider-azure \                
+ --start-sha 1cf1ec4a1effd9340fe7370ab45b173a4979dc8f  \
+  --end-sha e843409f896981185ca31d6b4a4c939f27d975de 
+  (commits range from first commit after previous release to new release commit)
+- Manually format and categorize
+
+## Promote image to prod repo	
+ Promote image (how is the image built?)
```

Comment:
(how is the image built?) --> https://testgrid.k8s.io/sig-cluster-lifecycle-cluster-api-provider-azure#post-cluster-api-provider-azure-push-images